application/src/main/java/org/togetherjava/tjbot/commands/tags/TagManageCommand.java

```diff
@@ -188,17 +199,31 @@ private void deleteTag(@NotNull CommandInteraction event) {
      * @param actionVerb the verb describing the executed action, i.e. <i>edited</i> or
      *        <i>created</i>, will be displayed in the message send to the user
      * @param event the event to send messages with, it must have an {@code id} option set
+     * @param subcommand the executed subcommand
+     * @param newContent the new content of the tag (@{@link Nullable}: only if assigning new
+     *        content)
```

Comment:
No need to link the nullable. just say `new content of the tag, or null if content is unchanged` 🤷

Reply:
done.